acc/views.py

```diff
@@ -10,13 +11,21 @@ def home_page(request):
         # Get user info
         name = request.POST["fname"]
         surname = request.POST["lname"]
+        email = request.POST["email"]
         career = request.POST["career"]
         bio = request.POST["about"]
 
+        # email details
         send_mail('Registration form', 
-         'First name: ' + name + '\n' + 'Last name: ' + surname + '\n' + 'Choice: ' + career + '\n' + 'About user: ' + bio,
+         'First name: ' + name + '\n' + 'Last name: ' + surname + '\n' + 'User email: ' + email + '\n' 'Choice: ' + career + '\n' + 'About user: ' + bio,
```

Comment:
Next time use f-sting, it is much easier to read and its fast
`print(f'First name: {name} \n Last Name: {surname}')`

Reply:
Got it!